app/Http/Controllers/CompetitionsController.php

```diff
@@ -37,18 +37,6 @@ public function __construct(UserRepositoryContract $repository, Manager $manager
     }
 
     /**
-     * Display a listing of the resource.
-     *
-     * @return \Illuminate\Http\Response
-     */
-    public function index()
```

Comment:
I mean that's the thing, I'm not sure we know yet how we want this to look/function? Removing it from the nav is fine, but I don't see a need to remove the method or view yet. We could always put it back in also, so no big deal either way.